src/app/backend/client/manager.go

```diff
@@ -281,12 +283,16 @@ func (self *clientManager) initConfig(cfg *rest.Config) {
 
 // Returns rest Config based on provided apiserverHost and kubeConfigPath flags. If both are
 // empty then in-cluster config will be used and if it is nil the error is returned.
-func (self *clientManager) buildConfigFromFlags(apiserverHost, kubeConfigPath string) (
+func (self *clientManager) buildConfigFromFlags(apiserverHost, apiserverHostCAFile, kubeConfigPath string) (
 	*rest.Config, error) {
-	if len(kubeConfigPath) > 0 || len(apiserverHost) > 0 {
+	if len(kubeConfigPath) > 0 || len(apiserverHost) > 0 || len(apiserverHostCAFile) > 0 {
```

Comment:
I guess we can ignore checking that and simply pass it to client config. Default value is empty string, so it will be ignored anyway. If someone will pass only `apiserverHostCAFile` without `apiserverHost` it will ignore it.

Reply:
:+1: